plugins/inputs/kubernetes/kubernetes.go

```diff
@@ -306,6 +308,15 @@ func buildPodMetrics(summaryMetrics *SummaryMetrics, podInfo []Metadata, labelFi
 				"container_name": container.Name,
 				"pod_name":       pod.PodRef.Name,
 			}
+			for k, v := range containerImages {
+				if k == container.Name {
+					tags["image"] = v
+					tok := strings.Split(v, ":")
+					if len(tok) == 2 {
+						tags["version"] = tok[1]
+					}
+				}
+			}
```

Comment:
Yes these are image and version tags. The reason why we need it is to identify the issues in a particular version/security or otherwise. Have a metric to identify fix. But anyways I just realized that the version is anyways part of the image, so I can skip it. Before the idea was to extract 2 tags - image and version, but now I can see that version is not required as extra tag. 

Is image tag fine as it helps us identify possible issues in a particular image?

Reply:
I wonder if splitting them is still a good idea. That way someone could drop the version if they wanted to or more easily filter on the container name without having to worry about the version.

What do you think?

Reply:
To be clear, do you mean container image name instead of container name? If yes, then It can be a good idea. Both tags can be present and it's up to users, if they want to discard version tag.

Reply:
> do you mean container image name instead of container name?

yes, sorry I should have been more specific.